OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -60,5 +60,12 @@ public void Resupply()
 		{
 			Self.QueueActivity(new ResupplyAircraft(Self));
 		}
+
+		[ScriptActorPropertyActivity]
+		[Desc("Returns true if the aircraft is flying circles.")]
```

Comment:
Can we add that this is a hacky workaround for our sucky activities and that people shouldn't rely on it existing in the long term, please?